### ISSUE TITLE
Index of ui options are not showing Terminal

### ISSUE THREAD

**Issue by imamhsn195** [LaravelDaily/Larastarters#35]
_**During installation in my terminal, it's showing something like the below picture.**_

 ![download](https://user-images.githubusercontent.com/44502980/141642508-08ca022c-144b-4995-9e98-77ad34e537d2.png)

**Comment by pcmehrdad** [comment]
Can you please try with another terminal (like putty or ...) and comment the results?

**Comment by PovilasKorop** [comment]
@imamhsn195 what is this terminal? It looks inside of some other tool or IDE?

**Comment by krekas** [comment]
Looks like vscode terminal and problem is with your system configuration

**Comment by imamhsn195** [comment]
@PovilasKorop Sorry, I did check repo issues for a few days.
It was VS Code integrated terminal.
I always work on this tool and last time I tried this repo to install the theme and I got this problem. The themes are installed and working only the number of options is not showing.

![141642508-08ca022c-144b-4995-9e98-77ad34e537d2](https://user-images.githubusercontent.com/44502980/142425043-7afcf43d-bb2b-425a-9b4d-5c5865db23b8.png)
g.
